### ISSUE TITLE
git installed with brew cannot exec

### ISSUE THREAD

**Issue by brandonkal** [Homebrew/homebrew-core#32541]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?
```
HOMEBREW_VERSION: 1.7.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: f239b7a62790cfb4f33bda9aab7597d43f1b1e3b
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: f3d5acb04536344cb10a844e639fc4f73c3bbe72
Core tap last commit: 4 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_GIT: git
HOMEBREW_MACOS_VERSION_NUMERIC: 101206
CPU: octa-core 64-bit sandybridge
Homebrew Ruby: 2.3.7 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/bin/ruby
Clang: 9.0 build 900
Git: 2.19.0 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_161
macOS: 10.12.6-x86_64
CLT: 9.2.0.0.1.1510905681
Xcode: 9.2
XQuartz: 2.7.11 => /opt/X11
```

To help us debug your issue please explain:
- What you were trying to do (and why)

Git installed via homebrew fails to exec aliases. MacOS git does not have this issue.

I would like to use git aliases for scripts, but somehow the homebrew installed git cannot do this.

- What happened (include command output)

1. `brew install git`
2. `sh -c "$(curl -fsSL https://r.viktoradam.net/githooks)"`
3. `cat ~/.gitconfig`
```
...
[alias]
	hooks = !~/.githooks/bin/githooks
...
```

4. cd to git repo
5. `git hooks disable`
Error:

```
fatal: cannot exec 'git-hooks': Permission denied
```

- What you expected to happen

The git alias would function

- Step-by-step reproduction instructions (by running `brew install` commands)

See above

**Comment by MikeMcQuaid** [comment]
> fatal: cannot exec 'git-hooks': Permission denied

You need to `chmod +x ~/.githooks/bin/githooks`. Do that and try again.

**Comment by brandonkal** [comment]
> > fatal: cannot exec 'git-hooks': Permission denied
> 
> You need to `chmod +x ~/.githooks/bin/githooks`. Do that and try again.

Yes, I should have made it clear that the scripts were made executable before opening this issue. Thanks.

I'm not sure why, but it seems that if you call the command from the Apple git, it then starts working with homebrew git.
Running this seemed to fix the problem:
```
/usr/bin/git hooks disable
```

After that, the git commands suddenly seem to work:
```
$ /usr/bin/git --version
git version 2.14.3 (Apple Git-98)
$ which git
/usr/local/bin/git
$ git hooks disable
<no permission errors>
$ git --version
git version 2.19.0
```
That may be something worth mentioning about installing git with homebrew.
